pkg/controller/operators/catalog/operator.go

```diff
@@ -882,6 +883,19 @@ func (o *Operator) syncCatalogSources(obj interface{}) (syncError error) {
 	return
 }
 
+func (o *Operator) isFailForwardEnabled(namespace string) (bool, error) {
```

Comment:
nit: does it make sense to pass the lister in as an argument and not have this function be a method? It doesn't really feel like checking if fail forward is enabled for a namespace should necessarily be part of the catalog operator versus a standalone function. This is more of a preference though, either way is fine.